src/brogue/Rogue.h

```diff
@@ -59,6 +59,9 @@
 // set to false to allow multiple loads from the same saved file:
 #define DELETE_SAVE_FILE_AFTER_LOADING  true
 
+// define to enable taking control of the player during playback
+// #define ENABLE_PLAYBACK_SWITCH
+
```

Comment:
I'd probably take this out, I see the macro being for clarity only, the only config is DEBUG=YES/NO. It causes errors if uncommented with debug=yes